src/utils/regexp.js

```diff
@@ -30,7 +30,7 @@ export function isLookbehindSupported() {
   return false;
 }
 
-export const HORIZONTAL_WHITESPACE = '[ \\t]';
+export const HORIZONTAL_WHITESPACE = '[ \\t ]';
```

Comment:
It would be better to use unicode code point `\u00a0` to match non-visible whitespaces.